packages/account/src/Sections/Profile/PersonalDetails/personal-details.jsx

```diff
@@ -34,10 +34,19 @@ import FormSubHeader from 'Components/form-sub-header';
 import FormSubmitErrorMessage from 'Components/form-submit-error-message';
 import LoadErrorMessage from 'Components/load-error-message';
 
-const removeObjProperties = (property_arr, { ...obj }) => {
-    property_arr.forEach((property) => delete obj[property]);
-    return obj;
-};
+const removeObjProperties = (property_arr, { ...obj }) =>
+    Object.fromEntries(
+        Object.entries(obj)
+            // eslint-disable-next-line no-unused-vars
+            .filter(([key, _]) => !property_arr.includes(key))
+    );
+
+const filterOnlyChangeableFields = (property_arr, { ...obj }) =>
```

Comment:
Would a name like `filterObjProperties` make more sense? since this function is not specific to changeable fields.